bundles/org.openhab.binding.heos/src/main/java/org/openhab/binding/heos/internal/handler/HeosChannelHandlerControl.java

```diff
@@ -79,4 +92,28 @@ private void handleCommand(Command command, String id) throws IOException, ReadE
                 break;
         }
     }
+
+    private boolean shouldPause(String id) {
+        Media applicableMedia = playingMediaCache.get(id);
+        if (applicableMedia == null || SONG.equals(applicableMedia.type)) {
+            return true;
+        } else {
+            // we have a station here, just have to check which one
+            switch (applicableMedia.sourceId) {
+                case 3: // Tunein
```

Comment:
Does it make sense in later changes to define the different Media Types (source IDs) as an Constant or ENUM? Just a question for the future.

Reply:
Yes, that makes a lot of sense, good idea, I updated it to use constants.